### DISCUSSION TITLE
Hyperaudio Lite as a plugin

### DISCUSSION THREAD

**Discussion by byskyphy** [hyperaudio/hyperaudio-lite-editor#236] [Ideas]
I've previously mentioned using hyperaudio lite audio as a plugin in Obsidian and/or Subtitle Edit #232 

[Subtitle Edit](https://github.com/SubtitleEdit/subtitleedit) is a fully featured, open source subtitle editor with a robust plugin ecosystem and plugin manager, that also allows to use locally run LLMs for Speech-to-Text recognition.

[Obsidian](https://obsidian.md/) is a very popular markdown-based text editor with an even more robust plugin ecosystem. 

Both are local and with a hyperaudio plugin, would allow to quickly ingest audio -> use TTS to locally generate a properly formatted hyperaudio file -> edit it as precisely as needed with dedicated tools -> export it in the correct format -> add them to a (Wordpress) website afterwards

**Comment by maboa** [comment]
Sorry for the slow response @alexvarha.

This sounds like an interesting approach. Although we don't have an Obsidian and/or Subtitle Edit plugin, we could look into this. BUt before we do I'd like to understand how this would differ to building our editor UI into the admin settings of our Wordpress Plugin? (Given we also have a caption editor).

**Reply by byskyphy** [reply]
I was unable to make your wordpress plugin work on my site at the time (in August). 

also, using local AI within subtitle edit's and obsidian's ecosystems would facilitate the adoption, as they'd allow users to keep their existing workflows, instead of relying on your editor's development rythm + wordpress + deepgram.
